pages/landing/index.tsx

```diff
@@ -0,0 +1,125 @@
+import Image from "next/image";
+import styles from "@/pages/landing/styles.module.scss";
+import landing_1 from "@/assets/images/landing/landing_1.png";
+import landing_2 from "@/assets/images/landing/landing_2.png";
+import landing_3 from "@/assets/images/landing/landing_3.png";
+import landing_4 from "@/assets/images/landing/landing_4.png";
+import landing_5 from "@/assets/images/landing/landing_5.png";
+import landing_6 from "@/assets/images/landing/landing_6.png";
+import landing_7 from "@/assets/images/landing/landing_7.png";
+import landing_10 from "@/assets/images/landing/landing_10.png";
+import landing_11 from "@/assets/images/landing/landing_11.png";
+import landing_12 from "@/assets/images/landing/landing_12.png";
+import Ellipse from "@/assets/icons/ic_Ellipse 22.svg";
+
+
+const Landing = () => {
+  return (
+    <>
+      <div className={styles["landing-container"]}>
+        <header className={styles["header"]}>
+          <section className={styles["wikid-banner"]}>
+            <div className={styles["banner-title"]}>
+              <div className={styles["paragraph"]}>남들이 만드는</div>
+              <div className={styles["heading"]}>나만의 위키</div>
```

Comment:
```suggestion
 <p className={styles["paragraph"]}>남들이 만드는</p>
              <p className={styles["heading"]}>나만의 위키</p>
```
`div`에서 `p` 태그 혹은 다른 적절한 태그(`<strong>` 등)를 사용해서 하는 게 더 좋을 것 같아요~! 다른 부분도 마찬가지입니다 ㅎㅎ

Reply:
아하   이게 의식한다고 했는데    잘안되네요!! 감사합니다.!